code/modules/core_implant/cruciform/rituals/priest.dm

```diff
@@ -216,6 +216,30 @@
 		if(target.wearer && target.wearer.stat != DEAD)
 			return target
 
+/datum/ritual/cruciform/priest/announcement
+	name = "Address the Flock"
+	phrase = "Cum autem magna conquisitio fieret, surgens Petrus dixit ad eos." //"After much discussion, Peter got up and addressed them"
```

Comment:
```suggestion
	phrase = "Postquam autem magna conquisitio fieret, surgens Petrus dixit ad eos."
```
I know its not a good translation but it removes a problematic word that leads to folks losing their mind in ooc/radio/ic

Reply:
This isn't a translation, this is a direct quote from the Latin Vulgate. People can laugh if they want, it's a common enough word in Latin.

Reply:
Please find a different quote then or have the quote be cut off to not have that word

Reply:
I changed the verse